Bottom: 581e7d94ce6b27fafaf42b30273a4cac2afec409
Top:    7c7ceecb105dda99ee8807b133213dc9bea8b36b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-22 23:57:26 +0000

Refresh of fixup-some-bugs-from-the-added

---

diff --git a/src/misc.c b/src/misc.c
index 31021c9..f60639e 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -2016,7 +2016,7 @@ MoveLayerToGroup (int layer, int group)
   int prev, i, j;
 
 #warning BUG NEXT LINE?
-  if (layer < 0 || layer > max_copper_layer + 1)
+  if (layer < 0 || layer >= max_copper_layer + 2)
     return -1;
   prev = GetLayerGroupNumberByNumber (layer);
   if ((layer == bottom_silk_layer
diff --git a/src/move.c b/src/move.c
index 5f0cfb3..9cdd49a 100644
--- a/src/move.c
+++ b/src/move.c
@@ -928,7 +928,6 @@ MoveLayer (int old_index, int new_index)
   AddLayerChangeToUndoList (old_index, new_index);
   IncrementUndoSerialNumber ();
 
-#warning BUG NEXT LINE?
   if (old_index < -1 || old_index >= max_copper_layer)
     {
       Message ("Invalid old layer %d for move: must be -1..%d\n",
